apps/web/src/components/forms/signin.tsx

```diff
@@ -221,7 +221,7 @@ export const SignInForm = ({ className, isGoogleSSOEnabled }: SignInFormProps) =
               onClick={onSignInWithGoogleClick}
             >
               <FcGoogle className="mr-2 h-5 w-5" />
-              Google
+              Sign In with Google
```

Comment:
Could you please revert the changes in this file?

Reply:
yeah sure. Done with it.

Though, would like to know the idea behind keeping it same.